docs/03-policies/prodev.md

```diff
@@ -0,0 +1,102 @@
+    * Having the courage to challenge the status quo, which leads to organizational learning
+    * Having the courage to be vulnerable, which leads to authentic communications and connection
+    * Open-minded thinking, which leads to increased tolerance of others ideas and personality styles
+
+CivicActions supports and encourages the Professional Development (prodev) of each team member by providing an annual budget of $1,200 per person to be used for prodev-related expenses.
+
+In addition to the $1,200 annual budget, each team member is encouraged to use a “take what you need” approach to using time for prodev pursuits during business hours. If you need guidance on what amount of time is appropriate to spend on prodev in a given year, consult the table below.
+
+Historical Prodev Hours Spent: 
+
+| Year  | Company-wide Prodev Hours Used | # People Billing Prodev Hours | Average Prodev Hours Used Per Person Annually |
+|-------|--------------------------------|-------------------------------|-----------------------------------------------|
+| 2015  | 599.25                         | 19                            | 31.5                                          |
+| 2016  | 963.5                          | 28                            | 34.4                                          |
+| 2017* | 1620                           | 35                            | 46.3                                          |
+ 
+_*As of Nov 21, 2017. All data was pulled from Harvest. [Link to data & calculations here.](https://docs.google.com/spreadsheets/d/12aWOg9I_nbolpaA9j4vbsv5ABZW2hYXfjkcRwRpTs1I/edit#gid=2143420373)_
+ 
+ ## Prodev Sprints
+ 
+ In order to provide some additional structure, support and focus to the prodev efforts of individuals, 1 month long prodev sprints will be held on a regular basis. During these sprints, participants will attend scrum meetings 2 to 3 times per week to talk about their prodev progress being made and connect with others pursuing their prodev goals at the same time. The sprint will also consist of a demo or review meeting, so that team members can share what they have accomplished or learned during the sprint. A retrospective will also take place so that the prodev sprint process can be continuously improved upon. 
+
+The total time commitment for participating in a prodev sprint will vary widely depending on what topic and intensity level each participant chooses to pursue, but the prodev sprint is not considered a full time project.
+
+Assignment to a prodev sprint is voluntary, and team members are encouraged to express their desire to participate to the prodev Practice Lead, Jason Heaster. Efforts will be made to group people with similar prodev goals or certification requirements. Once assigned to a sprint, participants’ project assignment schedules will be adjusted in Forecast to allocate time for the prodev sprint during that month.
```

Comment:
didn't realize this was becoming a practice. it makes sense to give it some focus, for sure.